README.md

```diff
@@ -94,13 +94,16 @@ Forking the repository for development
 ======================================
 
 1. Install [Go](https://golang.org)
-1. [Ensure your $GOPATH is set correctly](http://golang.org/cmd/go/#hdr-GOPATH_environment_variable)
-1. Install [godep](https://github.com/tools/godep)
+  * on Mac OS X this could be `brew install go` if you have [Homebrew](http://brew.sh/) installed.
+1. Create a directory for your source & dependencies ex. `mkdir ~/dev/cloudfoundry-dev && cd ~/dev/cloudfoundry-dev`
```

Comment:
I don't think there's any need to tell the user to `cd` here. It's not required by the rest of the instructions.